### ISSUE TITLE
Feature request: Word count of current block

### ISSUE THREAD

**Issue by akicebear** [lukeleppan/better-word-count#7]
My most frequent action is to select a block (paragraph) to asses what its word count is. I'd like to see an option that displays two word counts side by side in the status bar:
1. The current line word count OR selected text word count
2. The total word count

**Comment by lukeleppan** [comment]
I'm not too sure I understand. Would you like an additional word count that counts the words in the paragraph where the cursor is? It'll be toggleable.

**Comment by akicebear** [comment]
Correct, I'd like an additional word count indicator for the paragraph my cursor is currently within.

**Comment by lukeleppan** [comment]
Okay will release an update soon 👍

**Comment by Archie-2021** [comment]
I want something similar. i want a wordcount of text from start of the note to the current block (the cursor position) beside the one that show all the words in the note, it is very useful for measuring progress as i read books and papers in md.

thanks

**Comment by Alcky26** [comment]
I would like something similar, I use Templates a lot and every time I end up writing most of the content under only one part of the template. What i care about is the ammount of words in that specific area. It would be good if there was an option to put a Prefix like START: and a suffix like END and it automatically counts only the words inside that delimited area and nothing else on every note that has the prefix and suffix I chose. Of course, START: and END could be used in the text and that could be easily solved by adding an option to let the user decide which prefix and suffix to use as delimiters.

Thanks people.

**Comment by skaviouz** [comment]
When I was looking for a word count feature, I was likewise going to comment and add my thoughts, but I found an existing feature that indirectly solved my issue. I read all the comments, I'd like to point out that there is a "Section Word Count" Option now that displays the amount of words within an ATX Heading's Outline. Additionally, if there are child or sub-headings within the outline, It will also show a second amount based on the combined Word Count of both itself and all child ATX Headings.

![image](https://github.com/lukeleppan/better-word-count/assets/3837674/beff8c72-59bf-4aed-b144-6733b2fad379)

For Anyone else looking to make it easier to see, here is an example css, so I am using Wikipedia theme, so I change Wikipedia.css and append to the end of the file:  (I believe my 80% is 60% larger than the default of 50%.)

```
.bwc-section-count {
    font-size: 80%;
    color: green;
    font-weight: bold;
}
```

| Example | Output | Output with modified css |
| ------------- | ------------- | ------------- |
| # Example A (3/9)<br>Example A Text<br>## Example A-A (3)<br>Example A-A Text<br>## Example A-B (3)<br>Example A-B Text<br># Example B (7/18)<br>Example B Text is a little longer<br>## Example B-A (5)<br>Example B-A Text is different<br>## Example B-B (6)<br>Example B-B Text is uncreatively different | ![image](https://github.com/lukeleppan/better-word-count/assets/3837674/28910686-3f96-405f-ad8c-6290d10d1094) | ![image](https://github.com/lukeleppan/better-word-count/assets/3837674/140fa7d4-94df-45eb-9e3b-aecac72e7306)  |

If my hint wasn't direct enough, I feel that the numbers are too small, adding an option in the settings to make the text larger is my 2 cents.

**Comment by Endermanbugzjfc** [comment]
Any updates?
